.tool-versions

```diff
@@ -0,0 +1 @@
+erlang 19.2
```

Comment:
I wonder if there's a convention for these files. I had it my head that they were useful for projects/apps but maybe not utility libraries? I'm not sure. I'm not against it, but I wonder if there are conventions folks have though through. If there's an erlang version specified, should there also be an elixir?

Reply:
I can remove those (I always sneak them in), or we can add Elixir.  It only really matters if someone is using asdf.

Reply:
👍 on adding Elixir

Reply:
The other open source Elixir projects I've committed to do not have a `.tools-version` file or similar. So I think we should remove this.

Reply:
Sounds good.